modules/mobile/app/workers/mobile/v0/vaccines_updater_job.rb

```diff
@@ -0,0 +1,87 @@
+# frozen_string_literal: true
+
+module Mobile
+  module V0
+    # This job is run daily and pulls data from the CDC to create vaccine records
+    class VaccinesUpdaterJob
```

Comment:
TBH, I think this is a bit of a naive implementation. It simply goes through the first xml file (the one that has group names) and creates or updates records from those. If it encounters a new group name for a vaccine, it adds it to the vaccine record. The holes in this process are that 1) if there's ever a vaccine that's removed from the CDC xml, we don't delete it 2) if a group name is changed on the CDC xml, it doesn't remove the old group name, just adds the new one. 

These are both fixable issues, but it wasn't exactly clear how we want this to behave.